### DISCUSSION TITLE
how to modify the color of semantic tags on carla?

### DISCUSSION THREAD

**Discussion by LieYuan1993** [carla-simulator/carla#5201] [Q&A]
Hi everyone,
 I want to change some colors of semantic tags on Carla. I have found the CityScapesPalette.h and changed some values of the tags. But it did not work. Can someone tell me how to finish it?

Thank you in advance!

![1645606054(1)](https://user-images.githubusercontent.com/96498544/155286535-e441a29a-4080-4f01-805b-0092962f2eb3.png)

**Comment by NicolasPiques** [comment]
Same problem, did you resolve this issue?
I followed the "add semantic tags" tutorial but the color are unchanged when I launch Carla.

**Reply by IrammHamdard** [reply]
Did you find a solution to this issue ?

**Reply by NicolasPiques** [reply]
No, finally I have used a semantic tag an which already exists (other, unlabelled?). In my case I have deleted in my simulation all the objects except the one I want to only have my object with this label.
